docs/reference/hooks.rst

```diff
@@ -78,6 +78,24 @@ And here's an example of registering a hook function for a single block of code:
 
   # Hook is unregistered here
 
+If you need to register multiple hooks in a ``with`` block, you can pass the hooks in as a list of tuples:
+
+.. code-block:: python
+
+    def my_hook(...):
+        pass
+
+    def my_other_hook(...):
+        pass
+
+    with hooks.register_temporarily([
+            ('hook_name', my_hook),
+            ('hook_name', my_other_hook),
+        ]):
```

Comment:
You can make flake8 happy by un-indenting so that the closing bracket is in line with the `with`:
```suggestion
    with hooks.register_temporarily([
        ('hook_name', my_hook),
        ('hook_name', my_other_hook),
    ]):
```

Reply:
Interesting, I'm sure I tried this and it was unhappy!

Reply:
Seems to have worked, thanks!